model/docs/example_with_datasets.qmd

```diff
@@ -279,7 +279,7 @@ idata = az.from_numpyro(
 az.plot_lm(
     "negbinom_rv",
     idata=idata,
-)
+);
```

Comment:
I think the issue might be with capturing the output? In other plots we always do `out = plot`

Reply:
Is the issue with the website rendering? It appears good in my local machine. But I can't remember how to download the tar file for the website

Reply:
> Is the issue with the website rendering? It appears good in my local machine. But I can't remember how to download the tar file for the website

I think render to `md` works, but there is a problem with the `rst`. This is why I just opened https://github.com/CDCgov/multisignal-epi-inference/pull/250.